src/lib/flex/flex-align/flex-align.ts

```diff
@@ -20,29 +20,32 @@ import {
   MediaChange,
   MediaMonitor,
   StyleBuilder,
-  StyleDefinition,
-  StyleUtils
+  StyleBuilderOutput,
+  StyleUtils,
 } from '@angular/flex-layout/core';
 
 @Injectable({providedIn: 'root'})
-export class FlexAlignStyleBuilder implements StyleBuilder {
-  buildStyles(input: string): StyleDefinition {
-    const css: {[key: string]: string | number} = {};
+export class FlexAlignStyleBuilder extends StyleBuilder {
+  constructor() {
+    super();
+  }
+  buildStyles(input: string): StyleBuilderOutput {
+    const styles: {[key: string]: string | number} = {};
```

Comment:
Should we not define a type:

```ts
export type CSSStyle {
    [key: string]: string | number
}
```

and then use it

```ts
const styles : CSSStyle = { };
```

Reply:
We actually have a type! It's called `StyleDefinition`, and it wasn't used in the legacy code that I pulled. Refactored to use the actual type